src/components/header/index.tsx

```diff
@@ -15,6 +17,23 @@ const Header: FC = () => {
   const toggleMenu = () => {
     setIsSideBarOpen(!isSideBarOpen);
   };
+  const { currentUser, setCurrentUser } = useUserContext();
+
+  const handleChange = (name: string, value: string) => {
+    console.log(name);
```

Comment:
Please don't leave console logs in code or some debugging in code after it works.